deploy.py

```diff
@@ -664,6 +686,41 @@ def run(self):
         return 1
       return 0
 
+    if self.args.programmer == "bossac":
+      props = SUPPORTED_BOARDS[self.args.board]
+      # Call the Makefile in order to build and flash Tock OS using bossac
+      if self.args.tockos:
+        # The Arduino Nano requires a double tap on the reset button
+        # in order to enter bootloader mode
+        # We must wait for the user to confirm that
+        # the board is ready to load something on it
+        info("""Double tap the reset button on your Arduino Nano board
+            in order to begin the flashing process""")
+        info("Press [ENTER] when ready.")
+        _ = input()
+        self.checked_command(["make", "program"], cwd=props.path)
```

Comment:
Considering that at that point the kernel and the application are already compiled, why not writing both at the same time?
That's what we do for OpenOCD and DFU flashing by first calling
```py
self.create_hex_file(dest_file)
```
and then use the produced IntelHEX file to flash only once.

Reply:
Hello and thank you for the review! Yeah, this is a very good point, it would make more sense to create the hex file and flash only once. Regarding the changes, nano33ble does not support USB at all, so I've added that. Moreover, the board itself does not come with a button, which is why I've added the button to a certain GPIO (can change that, obviously). 
Compared to the upstream version, the nano33ble has some sensors which are not available on the tock version that OpenSK uses, which is why I've removed them for this scenario. (OpenSK does not require them anyways.)

So, in order to conclude, the board files which I've submitted have a different linker file (found some address mismatching when I tried to flash OpenSK, so I had to change that.), support for USB (which upstream does not have) and support for button(which upstream does not have)